clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/ops/DestroyAsgAtomicOperation.groovy

```diff
@@ -93,8 +94,17 @@ class DestroyAsgAtomicOperation implements AtomicOperation<Void> {
 
     if (autoScalingGroup.launchConfigurationName) {
       task.updateStatus BASE_PHASE, "Deleting launch config ${autoScalingGroup.launchConfigurationName} in $region."
-      autoScaling.deleteLaunchConfiguration(new DeleteLaunchConfigurationRequest(
-          launchConfigurationName: autoScalingGroup.launchConfigurationName))
+      try {
+        autoScaling.deleteLaunchConfiguration(
+          new DeleteLaunchConfigurationRequest(launchConfigurationName: autoScalingGroup.launchConfigurationName)
+        )
+      } catch (AmazonAutoScalingException e) {
+        if (e.message.toLowerCase().contains("launch configuration name not found")) {
+          // Ignore, because we are trying to delete it.
+        } else {
+          throw e
+        }
```

Comment:
Personal preference here, but, I'd switch it to look like so:

```
try {
  autoScaling.delete...
} catch (AmazonAutoScalingException e) {
  // ignore not found exceptions
  if (!e.message.toLowerCase().contains("launch configuration name not found")) {
    throw e;
  }
}
```

Reply:
Sure, I'll switch.